common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -53,7 +65,9 @@
         <input type="hidden" class="value" name="input_${id}" id="input_${id}" value="${value|h}" />
     % endif
 
-    <span class="status ${status.classname}" id="status_${id}" aria-describedby="label_${id}"><span class="sr">${status.display_name}</span></span>
+    <%include file="status_span.html" args="status=status,
```

Comment:
This problem type isn't supported, so my guess is that we don't really care if it has 2 status indicators, as long as they both show the correct thing.